src/Apps/Settings/Routes/Insights/Components/__tests__/InsightsMedianSalePrice.jest.tsx

```diff
@@ -0,0 +1,94 @@
+// test rending
+// test returning null
```

Comment:
👀

Reply:
These comments were for me 😂 forgot to remove them!